Core/source/entrypoint-bare.ts

```diff
@@ -21,7 +21,7 @@ global.handleRequestFromHost = (endpointName: string, request: string, data: str
       cb(formatBareOutput(fmtErr(new Error(`Unknown endpoint: ${endpointName}`))));
     } else {
       handler(JSON.parse(request), [Buf.fromBase64Str(data)])
-        .then(res => cb(formatBareOutput(Buf.concat(res))))
+        .then(res => cb(formatBareOutput(Buf.fromUtfStr(JSON.stringify(res)))))
```

Comment:
Here I would like to straight return the result back to Swift. So it would be `.then(res => cb(res))` which I guess is just `.then(cb)` ? And the type of `cb: (b64response: string) => void` will change to `cb: (res: EndpointRes) => void`.

That way, we return EndpointRes directly back to swift, which no longer has to parse it (after Swift is updated).